integration_tests/fixtures/forseti/modules/policy_library/templates/policy-library/lib/util.rego

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package validator.gcp.lib
+
+# has_field returns whether an object has a field
+has_field(object, field) {
+	object[field]
+}
+
+# False is a tricky special case, as false responses would create an undefined document unless
+# they are explicitly tested for
```

Comment:
nit: is this wrapping at 80-chars?  that's the convention we follow, so would be good to be consistent

Reply:
please update everywhere

Reply:
This is the template directly from the policy library. The policy library has it's own lint checker.